src/main/java/org/carlspring/maven/littleproxy/mojo/StopProxyMojo.java

```diff
@@ -37,15 +39,18 @@
      */
     @Parameter(property = "proxy.fail.if.not.running", defaultValue = "true")
     boolean failIfNotRunning;
+    
+    private static Logger LOGGER = LoggerFactory.getLogger(StopProxyMojo.class);
```

Comment:
@faisal-hameed : I know this breaks Sonar's convention, but could it be lowercase? All loggers across our codebase are lowercase.